client/src/components/ElementActions/tests/SaveAction-test.js

```diff
@@ -1,76 +1,79 @@
-  const WrappedComponent = (props) => <div>{props.children}</div>;
-  const ActionComponent = SaveAction(WrappedComponent);
-
-  it('renders a button when block is expandable', () => {
-    wrapper = mount(
-      <ActionComponent
-        title="My save action"
-        element={{
-          ID: 123,
-          BlockSchema: { type: 'Test' },
-          canCreate: true
-        }}
-        type={{ broken: false }}
-        expandable
-        actions={{ handlePublishBlock: mockMutation }}
-        toggle={false}
-      />
-    );
-    expect(wrapper.find('button').length).toBe(1);
-  });
+test('SaveAction renders a button when block is expandable', () => {
+  const { container } = render(
+    <ActionComponent {...makeProps()} />
+  );
+  expect(container.querySelector('button.element-editor__actions-save')).not.toBeNull();
+});
 
-  it('does not render a button when block is not expandable', () => {
-    wrapper = mount(
-      <ActionComponent
-        title="My save action"
-        element={{
-          ID: 123,
-          BlockSchema: { type: 'Test' },
-          canCreate: true
-        }}
-        type={{ broken: false }}
-        expandable={false}
-        actions={{ handlePublishBlock: mockMutation }}
-        toggle={false}
-      />
-    );
-    expect(wrapper.find('button').length).toBe(0);
-  });
+test('SaveAction does not render a button when block is not expandable', () => {
+  const { container } = render(
+    <ActionComponent {...makeProps({
+      expandable: false
+    })}
+    />
+  );
+  expect(container.querySelector('button.element-editor__actions-save')).toBeNull();
+});
 
-  it('does not render a button when block is broken', () => {
-    wrapper = mount(
-      <ActionComponent
-        title="My save action"
-        element={{
-          ID: 123,
-          BlockSchema: { type: 'Test' },
-          canCreate: true
-        }}
-        type={{ broken: true }}
-        expandable
-        actions={{ handlePublishBlock: mockMutation }}
-        toggle={false}
-      />
-    );
-    expect(wrapper.find('button').length).toBe(0);
-  });
+test('SaveAction does not render a button when block is broken', () => {
+  const { container } = render(
+    <ActionComponent {...makeProps({
+      type: {
+        broken: true
+      }
+    })}
+    />
+  );
+  expect(container.querySelector('button.element-editor__actions-save')).toBeNull();
 });
+
+//   it('does not render a button when block is broken', () => {
+//     wrapper = mount(
+//       <ActionComponent
+//         title="My save action"
+//         element={{
+//           ID: 123,
+//           BlockSchema: { type: 'Test' },
+//           canCreate: true
+//         }}
+//         type={{ broken: true }}
+//         expandable
+//         actions={{ handlePublishBlock: mockMutation }}
+//         toggle={false}
+//       />
+//     );
+//     expect(wrapper.find('button').length).toBe(0);
+//   });
+// });
```

Comment:
I'm not comfortable having tests that are just commented out. There is clear evidence in other places in the codebase of "todo" comments and commented out code that were meant to be resolved later that just never got resolved. Having commented out tests is the same - we may intend to get back to it, but it's entirely possible if not likely that the task for doing so will just keep being deprioritised until it's forgotten entirely.

@maxime-rainville I'll defer to your judgment on this as mentioned in standup.